src/main.cpp

```
@@ -426,20 +427,26 @@ bool SMPDebug_Execute(
 		auto parseValidateModeArg = [&](const char* arg) {
 			if (arg[0] == '\0')
 				return true;
 			if (_stricmp(arg, "gear") == 0) {
 				gearOnly = true;
 				return true;
 			}
+			if (_stricmp(arg, "warnings") == 0) {
+				includeWarnings = true;
+				return true;
+			}
 			if (_stricmp(arg, "error") == 0) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)